.github/actions/staging-api-deploy/action.yml

```diff
@@ -90,7 +146,7 @@ runs:
       with:
         task-definition: task-definition.json
         container-name: nginx
-        image: "ghcr.io/wordpress/openverse-api_nginx:${{ inputs.tag }}"
+        image: "ghcr.io/wordpress/openverse-api-nginx:${{ inputs.tag }}"
```

Comment:
This was correct earlier. The image on GHCR: https://github.com/WordPress/openverse/pkgs/container/openverse-api_nginx